examples/remove_widgets_example.py

```diff
@@ -10,8 +10,7 @@ def __init__(self, parent=None):
         QtWidgets.QMainWindow.__init__(self, parent)
 
         # create a FormDockWidget
-        dock = UIFormWidget.FormDockWidget(parent=self)
-        dock.setWindowTitle('Example remove widget')
+        dock = UIFormWidget.FormDockWidget(parent=self, title='Example remove widget')
```

Comment:
I tested this example with changing this line to remove the 'title' argument.
It failed:
```
Traceback (most recent call last):
  File "c:\Users\lhe97136\Work\eqt\examples\remove_widgets_example.py", line 127, in <module>
    window = MainUI()
             ^^^^^^^^
  File "c:\Users\lhe97136\Work\eqt\examples\remove_widgets_example.py", line 13, in __init__
    dock = UIFormWidget.FormDockWidget(parent=self)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\lhe97136\AppData\Local\mambaforge\envs\eqt\Lib\site-packages\eqt\ui\UIFormWidget.py", line 340, in __init__
    QtWidgets.QDockWidget.__init__(self, title, parent)
TypeError: 'PySide2.QtWidgets.QDockWidget.__init__' called with wrong argument types:
  PySide2.QtWidgets.QDockWidget.__init__(NoneType, MainUI)
Supported signatures:
  PySide2.QtWidgets.QDockWidget.__init__(typing.Optional[PySide2.QtWidgets.QWidget] = None, PySide2.QtCore.Qt.WindowFlags = Default(Qt.WindowFlags))
  PySide2.QtWidgets.QDockWidget.__init__(str, typing.Optional[PySide2.QtWidgets.QWidget] = None, PySide2.QtCore.Qt.WindowFlags = Default(Qt.WindowFlags))
```

Reply:
Could add a unit test for successful creation of a FormDockwidget with and without this title parameter being set?

Reply:
I added unit test to check the FormDialog and FormDockWidget with default title, title=None and title: str